app/frontend/src/components/views/workspace-view/WorkspaceView.tsx

```diff
@@ -0,0 +1,72 @@
+import { $workspace, getWorkspaceFx } from 'store/workspaces';
+import { useStore } from 'effector-react';
+import React, { useEffect } from 'react';
+import Sidebar from '../../layouts/base/Sidebar';
+import SidebarHeader from '../../layouts/base/SidebarHeader';
+import ProjectList from '../../UI/project-list/ProjectList';
+import { Link, Route, Routes } from 'react-router-dom';
+import Content from '../../layouts/base/Content';
+import WorkspaceForm from '../workspace-form/WorkSpaceForm';
+import ProjectForm from '../project-form/ProjectForm';
+import ProjectView from '../project-view/ProjectView';
+import ColorVariables from 'styles/Colors';
+import GlobalStyles from 'styles/Global';
+import Container from '../../layouts/base/Container';
+import styled from 'styled-components';
+import Button from '../../UI/button/Button';
+
+/**
+ * Props of the component
+ */
+interface Props {}
+
+/**
+ * WorkspaceView component
+ */
+const WorkspaceView: React.FC<Props> = () => {
+  const workspace = useStore($workspace);
+
+  useEffect(() => {
+    getWorkspaceFx({});
+  }, []);
+
+  return (
+    <Container>
+      <ColorVariables/>
```

Comment:
I would suggest leaving this content in `App` component as it was before. And just importing workspace store where it's needed (like inside of `WorkspaceForm` component)